tags.go

```diff
@@ -0,0 +1,31 @@
+package distribution
+
+import (
+	"github.com/docker/distribution/context"
+	"github.com/docker/distribution/reference"
+)
+
+// TagService provides access to information about tagged objects.
+type TagService interface {
```

Comment:
Currently a repository returns an instance to its tag service.   In the future there will be a tag service.  docker notary for instance.